classes/controllers/deadline_extensions_controller.php

```diff
@@ -220,19 +227,19 @@ protected function update_deadline_extension() {
         $ability->require_can('update', $this->deadlineextension);
 
         if ($this->form->is_validated()) {
-            $values->extra_information_text = $values->extra_information['text'];
-            $values->extra_information_format = $values->extra_information['format'];
+            $values->extrainformationtext = $values->extra_information['text'];
+            $values->extrainformationformat = $values->extra_information['format'];
             $this->deadlineextension->update_attributes($values);
 
-            $personaldeadline = personal_deadline::get_personal_deadline_for_student(
+            $personaldeadline = personaldeadline::get_personaldeadline_for_student(
```

Comment:
again, this is weird either keep all the _ or get rid of all of them. keep them all is more readable imho